diff -up plasma-workspace-5.21.0/ksplash/ksplashqml/themes/Classic/main.qml.1~ plasma-workspace-5.21.0/ksplash/ksplashqml/themes/Classic/main.qml
--- plasma-workspace-5.21.0/ksplash/ksplashqml/themes/Classic/main.qml.1~	2021-02-11 13:20:10.000000000 +0100
+++ plasma-workspace-5.21.0/ksplash/ksplashqml/themes/Classic/main.qml	2021-02-14 22:43:51.903785127 +0100
@@ -52,7 +52,7 @@ Item {
     Image {
         id: background
         anchors.fill: parent
-        source: "images/background.png"
+        source: "/usr/share/mdk/backgrounds/OpenMandriva-splash.png"
     }
 
     Image {
diff -up plasma-workspace-5.21.0/lookandfeel/contents/components/WallpaperFader.qml.1~ plasma-workspace-5.21.0/lookandfeel/contents/components/WallpaperFader.qml
--- plasma-workspace-5.21.0/lookandfeel/contents/components/WallpaperFader.qml.1~	2021-02-11 13:20:10.000000000 +0100
+++ plasma-workspace-5.21.0/lookandfeel/contents/components/WallpaperFader.qml	2021-02-14 22:43:51.903785127 +0100
@@ -114,11 +114,11 @@ Item {
             }
             PropertyChanges {
                 target: wallpaperFader
-                factor: 1
+                factor: 0
             }
             PropertyChanges {
                 target: clock.shadow
-                opacity: 0
+                opacity: 1
             }
             PropertyChanges {
                 target: clock
